_sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md

```diff
@@ -9,9 +9,9 @@ Taken together, making safer every-day choices on average --- not just this week
 -   Non-contact sports are better than contact ones. E.g., Tennis is better than (\>) rugby
 -   Takeout \> eat on restaurant patio \> eat in restaurant
 -   Video streaming options \> movie theatre
--   Outside events \> inside events (small events also are better than large ones)
+-   No group events \> Outside events \> inside events (small events also are better than large ones)
 -   Delivery/curbside pickup \> going in person
 -   Remote observance \> religious service attendance
 -   Birthday cupcakes \> birthday cake (if candle blowing is involved)
--   Drink your coffee/beer/cocktail at a table instead \> drink your coffee/beer/cocktail at the bar
+-   Take your coffee/beer/cocktail to go \> drink your coffee/beer/cocktail at the bar
 -   Live performers are going to be hard hit economically. Consider supporting their work on Patreon, directly via PayPal, or attend only the smaller shows. Promote them on social media. Buy their work directly.
```

Comment:
I think shows are cancelled everywhere, yeah?

Reply:
There are still live shows in Mexico, and there could be other places where we have potential users as well. I'd rather we just delete the part about "attend only the smaller shows" as that seems like something we shouldn't be endorsing given how crowded even smaller shows can get.

Reply:
I agree to delete

Reply:
I think only @mellybelly can accept that deletion on this PR, but I'm not sure she saw my note below. I'll merge this PR and then do the mods immediately after.